src/client/content_cao.cpp

```diff
@@ -917,11 +926,6 @@ void GenericCAO::updateNodePos()
 
 void GenericCAO::step(float dtime, ClientEnvironment *env)
 {
-	if (m_animated_meshnode) {
```

Comment:
Since the positions of bones depend on the position of the model, the skeleton must be processed last. So this block is now at the end of step.